modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java

```diff
@@ -161,6 +167,24 @@ else if (cmd.equals(Constants.UPDATE)) {
 						KBArticle.class, kbArticle.getResourcePrimKey(),
 						redirect));
 			}
+
+			if (kbArticle.isScheduled()) {
+				Format dateTimeFormat = FastDateFormatFactoryUtil.getDateTime(
+					themeDisplay.getLocale());
+
+				MultiSessionMessages.add(
+					actionRequest, "kbArticleScheduledSuccessMessage",
+					_language.format(
+						_portal.getHttpServletRequest(actionRequest),
+						"x-will-be-published-on-x",
+						new Object[] {
+							"<strong>`" + HtmlUtil.escapeURL(title) +
```

Comment:
Is this `escapeURL` method correct? 
Maybe `HtmlUtil.escapeAttribute(title)` is better?

Reply:
I used this method because it's used as on **EditFileEntryMVCActionCommand**.
I really do not understand the difference between `escapeURL` and `escapeAttribute` when it's applied on the toast.

If escapeAttribute is better I will change it.

Thanks!

Reply:
Maybe only `HtmlUtil.escape` but I´m not sure

![image](https://github.com/liferay-lima/liferay-portal/assets/67901/882d7355-3745-4d2f-afab-1d4d69e50f00)

Reply:
I go with `HtmlUtil.escape(title)` some test

xss ✅ 
![image](https://github.com/liferay-lima/liferay-portal/assets/67901/7c53eea8-2ba3-4575-abb8-5773d05a9d76)

spaces and special chars ✅ 
![image](https://github.com/liferay-lima/liferay-portal/assets/67901/f6776d92-a3b3-47bb-955c-dca186db5bbb)